src/xinterpreter_wasm.cpp

```diff
@@ -17,7 +17,13 @@ namespace xcpp
 {
 
     wasm_interpreter::wasm_interpreter()
-        : interpreter(0, nullptr)
+        : interpreter(create_args().size(), create_args().data())
     {
     }
+
+    std::vector<const char*> wasm_interpreter::create_args()
+    {
+        static std::vector<const char*> Args = {"-Xclang", "-std=c++20"};
+        return Args;
+    }
```

Comment:
Currently we are only presenting an xcpp20 kernel for the users to use by passing the c++ version as 20 to our wasm interpreter. This would be fixed in the future as raised in the TODO.